app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str):
+    check_list = command.split(" ")
+    if check_list[1] != check_list[2]:
+        with open(f"{check_list[1]}", "r") as file_in:
```

Comment:
no need to use f-formatting, check_list[1] is a string anyway
use just check_list[1]